src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,515 @@
+
+  Learn more about logs [here](/docs/logs/get-started/get-started-log-management/).
+
+  </Side>
+  </SideBySide>
+
+# Network [#network]
+<SideBySide>
+  <Side>
+
+  With <InlinePopover type="networkMonitoring" />, you can analyze how well your routers, switches, and other networking devices perform. It will also build a visual network topology, letting you find performance problems in your environment. 
+  
+  Learn more about network [here](/docs/network-performance-monitoring/get-started/npm-introduction/).
+
+  </Side>
+  <Side>
+
+<img
+  title="New Relic network"
+  alt="New Relic network"
+  src={NRNetworkMain}
+/>
+
+  </Side>
+  </SideBySide>
+
+# Mobile [#mobile]
+<SideBySide>
+  <Side>
+<img
+  title="New Relic mobile"
+  alt="New Relic mobile"
+  src={NRMobileMain}
+/>
+
+  </Side>
+  <Side>
+
+  The <InlinePopover type="mobile" /> capability sends telemetry directly from the end user's mobile app to New Relic. This allows us to analyze the app's performance on the user's device. This capability helps with the performance and crash troubleshooting of your Android, iOS, or hybrid mobile apps. Use mobile monitoring to improve your app's user experience. Or, study HTTP and network performance to work better with your backend teams.
+
+  Learn more about mobile [here](/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-mobile-monitoring/).
+
+  </Side>
+  </SideBySide>
+
+# OpenTelemetry [#opentelemetry]
+<SideBySide>
+  <Side>
+
+  OpenTelemetry helps collect data from your applications and hosts, sending it to New Relic. New Relic's platform lets you analyze the data and fix any application problems. This toolkit simplifies the process of monitoring and troubleshooting your software. By utilizing OpenTelemetry, you can enhance the performance and reliability of your applications.
+
+  Learn more about OpenTelemetry [here](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-get-started-intro/).
+
+  </Side>
+  <Side>
+
+<img
+  title="New Relic OpenTelemetry"
+  alt="New Relic OpenTelemetry"
+  src={NROpenTelemetryMain}
+/>
+
+  </Side>
+  </SideBySide>
+
+# Serverless and Lambda [#serverless-lambda]
+<SideBySide>
+  <Side>
+
+<img
+  title="New Relic Lambda functions"
+  alt="New Relic Lambda functions"
+  src={NRLambdaMain}
+/>
+
+  </Side>
+  <Side>
+
+  We offer in-depth performance monitoring for your serverless AWS Lambda functions. Our Lambda layer gives one view into the details of your Lambda functions. You can understand what's happening in your serverless apps. Lambda monitoring is serverless and uses CloudWatch data and code-level tools to give you more in-depth data. 
+
+  Learn more about serverless and Lambda [here](/docs/serverless-function-monitoring/aws-lambda-monitoring/get-started/monitoring-aws-lambda-serverless-monitoring/).
+
+  </Side>
+  </SideBySide>
+
+# Service levels [#service-levels]
```

Comment:
IMO if we're giving service levels an H2, then i think that NRQL and NRDB need to be their own H2s as well. Service levels are more of a conceptual _practice_ rather than a capability per se, and you create them with NRQL too

Reply:
I've created a section for `Query your data` (NRQL), but I don't think we should also add a section for NRDB because it's not a capability inside the product, I believe. We're adding, on this page, a brief overview of our most popular capabilities.